test/object-store/util/sync/sync_test_utils.cpp

```diff
@@ -507,39 +507,16 @@ void wait_for_num_objects_in_atlas(std::shared_ptr<SyncUser> user, const AppSess
         std::chrono::minutes(15), std::chrono::milliseconds(500));
 }
 
-void trigger_client_reset(const AppSession& app_session)
+void trigger_client_reset(const AppSession& app_session, const SyncSession& sync_session)
```

Comment:
don't we need to set development mode back and wait for initial sync to complete as before? (is the client reset endpoint doing all that?)

Reply:
The client reset endpoint just directly triggers a client reset for a specific client file ident without disabling sync, so none of that is needed.